cpp/data/system_updates/147.ixtheo.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE keyword_translations ADD COLUMN translation_disabled TINYINT(1) AFTER priority_entry;
```

Comment:
Should we add e.g. DEFAULT 0 here? The field does not seem to be covered in the initial INSERT statement.